packages/client/src/client4.ts

```diff
@@ -2138,6 +2138,15 @@ export default class Client4 {
         );
     }
 
+    addPostReminder = (userId: string, postId: string, timestamp: number) => {
+        this.trackEvent('api', 'api_post_set_reminder');
```

Comment:
is this new telemetry key made aware to Telemetry team?

Reply:
@sri-byte Please let me know, do I need to change anything here